suite/tests/CMakeLists.txt

```diff
@@ -3079,11 +3083,21 @@ if (BUILD_SAMPLES)
         endif (UNIX)
       elseif (sample STREQUAL "memval_simple")
         # i#2449: only witnessed crash on 32-bit linux
-        if (X86 AND (NOT X64) AND UNIX)
-          tobuild(client.memval-test client-interface/memval-test.c)
-          torunonly_ci(sample.${sample} client.memval-test ${sample}
-              client-interface/memval-test.c "" "" "")
-        endif(X86 AND (NOT X64) AND UNIX)
+        if (X86 AND UNIX)
+          if (NOT X64)
```

Comment:
Does this memval-test run on 64-bit?  If so it seems best to just remove the `NOT X64` and the `i#2449` comment.

Reply:
It works. Done.